tests/expressions_case/tests.py

```diff
@@ -1165,6 +1165,16 @@ def test_m2m_reuse(self):
             lambda x: x[1:]
         )
 
+    def test_empty_q_object(self):
+        """A Q object with an empty condition should be rejected as the
+        conditional expression in a Case()."""
+        with self.assertRaisesMessage(ValueError, 'Empty Q() object'):
```

Comment:
Please include the entire message.

Reply:
Hmm. I'm not really sure what that achieves, other than making the test (slightly) more brittle. But if that's the coding standard then it's not worth arguing over. I've updated the patch, anyway.